src/main/protobuf/extvm/msg.proto

```diff
@@ -0,0 +1,142 @@
+syntax = "proto3";
+package io.iohk.ethereum.extvm;
+
+message CallContext {
+  bytes ownerAddr = 1;
```

Comment:
Owner's address determines the context account for state modifications, e.g. this is an address for which storage modifications will happen by executing `SSTORE`. 

The recipient's address is an implementation artifact, as for calls resulting from directly transactions (VM @ depth 0), it will always be the same as owner's address, thus will be removed from the final version.

(Internally we needed to distinguish owner's and recipient's addresses to correctly handle _precompiled contracts_, because of the possibility to call them using `DELEGATECALL` and `CALLCODE`, and how those instructions affect owner's address.)

Reply:
Yes, but again, I see no reason to keep it in the final version.